src/librustc/session/config.rs

```diff
@@ -492,6 +493,14 @@ impl Options {
         self.incremental.is_none() ||
         self.cg.codegen_units == 1
     }
+
+    pub fn file_path_mapping(&self) -> FilePathMapping {
+        FilePathMapping::new(
+            self.debugging_opts.remap_path_prefix_from.iter().zip(
+                self.debugging_opts.remap_path_prefix_to.iter()
+            ).map(|(src, dst)| (src.clone(), dst.clone())).collect()
```

Comment:
Given that the values are paths, and most file systems do not like colon `':'` in path names, can't a single argument be bifurcated on a colon? Example: `-Z remap-path="original_path_from:preferred_path_to"`.